src/controllers/handlers/explorer-handler.ts

```diff
@@ -41,10 +41,12 @@ export type MixedWearableResponse = Omit<MixedWearable, 'minTransferredAt' | 'ma
 
 export type MixedWearableTrimmedResponse = {
   entity: ExplorerWearableEntity
+  amount?: number
 }
 
 export type WearableFilters = {
   isSmartWearable: boolean
+  isPolygonWearable: boolean
```

Comment:
can we use `network` here? I believe it's more flexible and scalable to use `network`, wdyt?